### PR TITLE
Renovation: add the ability to enable renovation in the qunit tests

### PR DESCRIPTION
**Pull request by churkin**
image

### CODE REVIEW

File: js/core/config.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const config = {
     forceIsoDateParsing: true,
     wrapActionsBeforeExecute: true,
     useLegacyStoreResult: false,
+    useRenovatedComponents: false,
```
Comment (churkin) [inline js/core/config.js:24]: @RomanTsukanov check this new private field name in the global config pls.

File: testing/helpers/qunitExtensions.js [hunk 2]
```diff
@@ -170,6 +170,22 @@
     });
 })();
 
+(function enableRenovation() {
+    QUnit.config.urlConfig.push({
+        id: 'enablerenovation',
+        label: 'Enable renovation',
+        tooltip: 'Turn on to use renovated components instead of outdated.'
```
Comment (churkin) [inline testing/helpers/qunitExtensions.js]: @RomanTsukanov and the tooltip's text

File: js/core/config.js (hunk 1 above)
Comment (RomanTsukanov) [reply js/core/config.js:24]: This one is fine.

File: testing/helpers/qunitExtensions.js (hunk 2 above)
Comment (RomanTsukanov) [reply testing/helpers/qunitExtensions.js]:
```suggestion
        tooltip: 'Select to use renovated components'
```

**Comment by churkin** [comment]
Ready to review

**Pull request closed without merging**